package.json

```diff
@@ -32,7 +32,7 @@
     "body-parser": "^1.19.0",
     "ddk.registry": "git+https://github.com/AraiEzzra/DDK.REGISTRY.git",
     "dotenv": "^8.0.0",
-    "eska-common": "^1.0.7",
+    "eska-common": "=1.0.7",
```

Comment:
I suspect, that "=" is redundant
![Screenshot from 2019-11-14 18-07-37](https://user-images.githubusercontent.com/19222673/68874431-df428480-0709-11ea-83b1-2cd00fc04087.png)